elasticsearch/src/main/scala/com/snowplowanalytics/snowplow/loader/clients/ElasticsearchSenderHTTP.scala

```diff
@@ -93,19 +93,19 @@ class ElasticsearchSenderHTTP(
       futureToTask(client.execute(bulk(actions)))
         // we rety with linear backoff if an exception happened
         .retry(delays, exPredicate(connectionAttemptStartTime))
-        .map { bulkResponse =>
-          bulkResponse.items.zip(records)
+        .map {{
+          case Right (bulkResponseResponse) =>  bulkResponseResponse.result.items.zip(records)
             .map { case (bulkResponseItem, record) =>
               handleResponse(bulkResponseItem.error.map(_.reason), record)
             }.flatten
-        }.attempt.unsafePerformSync match {
-          case \/-(s) => s.toList
-          case -\/(f) =>
-            log.error(s"Shutting down application as unable to connect to Elasticsearch for over $maxConnectionWaitTimeMs ms", f)
-            // if the request failed more than it should have we force shutdown
-            forceShutdown()
-            Nil
-        }
+        }}.attempt.unsafePerformSync match {
+        case \/-(s) => s.toList
```

Comment:
indent seems weird here

Reply:
My intellij auto-format doesn't really work apparently. I will format it manually.